runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java

```diff
@@ -54,11 +54,22 @@ default int lapTimeout()
 		return 5;
 	}
 
+	@ConfigItem(
+		keyName = "highlightShortcuts",
+		name = "Highlight Agility Shortcuts",
+		description = "Enable/disable the highlighting of Agility shortcuts",
+		position = 3
```

Comment:
It looks like this is moved with no other changes? What's the purpose behind that?

Reply:
I had added a config, and organised the config to fit the new fields. It was requested that the config be removed, but I didn't revert the whole file. I'll do that though 👍